zkevm-circuits/src/circuit_tools/cell_manager.rs

```diff
@@ -103,11 +103,11 @@ impl<F: Field> Expr<F> for &Cell<F> {
     }
 }
 
-pub(crate) type WordCell<F> = Word<Cell<F>>;
+pub(crate) type WordHiLoCell<F> = WordHiLo<Cell<F>>;
 
-impl<F: Field> WordCell<F> {
-    pub fn expr(&self) -> Word<Expression<F>> {
-        Word::new([self.lo().expr(), self.hi().expr()])
+impl<F: Field> WordHiLoCell<F> {
+    pub fn expr(&self) -> WordHiLo<Expression<F>> {
+        WordHiLo::new([self.lo().expr(), self.hi().expr()])
```

Comment:
I notice a problem naming it `WordHiLo`. 
We put the first argument "Low" and the second argument "High," which is the reverse order of "HiLo."
I propose we name it `WordLoHi` to reflect the correct order of the argument.
Or even we can drop the `Word` and call it `LoHi` for brevity. And it seems "Lohi" is [salmon in Finnish](https://fi.wikipedia.org/wiki/Lohi).

Reply:
Yes thanks for pointing out, LoHi is more logical.

> And it seems "Lohi" is [salmon in Finnish](https://fi.wikipedia.org/wiki/Lohi).

😄